gguf-py/gguf/gguf.py

```diff
@@ -85,6 +85,7 @@ class MODEL_ARCH(IntEnum):
     GPTNEOX       : int = auto()
     MPT           : int = auto()
     STARCODER     : int = auto()
+    PERSIMMON         : int = auto()
```

Comment:
```suggestion
    STARCODER     : int = auto()
    PERSIMMON     : int = auto()
```

Reply:
added all the formatting changes, thanks for catching!